packages/react-component-playground/src/components/component-playground.jsx

```diff
@@ -359,21 +370,10 @@ module.exports = React.createClass({
         fixtureChange: this.state.fixtureChange + 1
       }));
     }
-
-    // Focus on the editor when changing fixture, to prevent overriding
-    // its contents with the state generated by the initial unfolding of the
-    // rendered component tree
-    if (this.props.editor) {
-      this._focusOnEditor();
-    }
-  },
-
```

Comment:
@skidding This was causing some trouble, as focusing the editor on fixture click would only provide it with the fixture contents. `onFixtureUpdate` wouldn't do anything, because it was focused. Then, on editor blur, it would also add the previewed component's serialization (things such as `state`, for example) and it looked pretty bad.

Are you ok with removing it? Maybe I misunderstood it.